bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -355,6 +355,13 @@
    @XmlElement
    private int                   tourAltDown;
 
+
+   /**
+    * Average altitude change (m/km)
+    */
+   @XmlElement
+   private int                   tourAvgAltChange;
```

Comment:
The field `tourAvgAltChange` should have a better name.

In the beginning of MT all fields started with `tour...` however all fields are just for one tour in `TourData` and `Alt` should be named without abbreviation to be easier to read. `Avg` is OK as it is commonly used.

So my suggestion is `avgAltitudeChange`

Reply:
Done (see PR)